index.rst

```diff
@@ -6,24 +6,498 @@
+=========================================================
+
+The following applies when creating or updating tutorials in the `dp0-2_lsst_io repository <https://github.com/lsst/dp0-2_lsst_io>`_, which is part of the ``lsst`` GitHub Organization.
+
+Develop new tutorials, or update existing ones, in a new branch.
+This branch should be named for the corresponding Jira ticket (e.g., "tickets/PREOPS-12345").
+The new branch should be created from ``main``.
+Typically, only one tutorial is updated per ticket branch.
+
+Make commits and push changes to your branch in the ``dp0-2_lsst_io`` repository until work is complete, then open a pull request to ``main``.
+
+Contact one or more Rubin Observatory staff members with the appropriate expertise and ask them to review the notebook.
+At least one reviewer should be a member of the CET.
+If they agree, assign them as a reviewer on your pull request.
+
+Ensure that all of the reviewers' comments are addressed.
+Make changes and new commits to the branch, and respond to all of their comments with either a confirmation a change was made,
+or an explanation of why the request was not implemented.
+
+Contact the reviewers to let them know the pull request now awaits their approval.
+
+After the reviewers have approved the pull request, ``rebase and merge`` your ticket branch into the ``main`` branch.
+Resolve all conflicts, if there are any.
+After the successful merge, delete your branch.
+
+Remember to make comments in the associated Jira tickets about the major updates and mark the ticket as done.
+
+
+Stretch goals
+=============
+
+Work is on-going in these areas.
+
+Notebook metadata
+-----------------
+
+Embed notebook metadata (e.g., version, skills, packages) in a way that can be scraped and used to auto-generate the ``README.md`` file or a Table of Contents, to enable users to browse notebook contents.
+
+Narrative voice
+---------------
+
+Adopt a best practice for narrative instructions: should we use terms like "you," or should a passive voice be adopted?
+
+Accessibility
+-------------
+
+Improve tutorials' accessibility to people with visual disabilities by finding and implementing, e.g., screen reader compatibility software, data sonification packages.
+
+Translations
+------------
+
+At minimum, translate any undergraduate-level tutorials into Spanish.
+
+Additionally, improve tutorials' accessibility to non-English speakers by finding and implementing automatic translation and localization software.
+
+Purge extraneous items in notebooks
+-----------------------------------
+
+Develop a best practice for how to keep notebook memory usage in check, in addition to deleting figures.
+Do not rely on the ``del`` command for this.
+
+Recipe functions
+----------------
+
+Create recipes for common user activities.
+These could be, e.g., ADQL searches for the portal, code snippets for the command line, or python modules that can be imported.
```

Comment:
If we develop python modules, then we'll need to figure out how/where to make those available?